pkg/cmd/cli/describe/printer.go

```diff
@@ -34,7 +34,7 @@ var (
 	imageColumns            = []string{"NAME", "DOCKER REF"}
 	imageStreamTagColumns   = []string{"NAME", "DOCKER REF", "UPDATED", "IMAGENAME"}
 	imageStreamImageColumns = []string{"NAME", "DOCKER REF", "UPDATED", "IMAGENAME"}
-	imageStreamColumns      = []string{"NAME", "DOCKER REPO", "TAGS", "UPDATED"}
+	imageStreamColumns      = []string{"NAME", "TAGS", "UPDATED"}
```

Comment:
Again, everything in the original issue #5927. Generally, the idea is that in most cases this represents the internal registry which is confusing, because in all cases it points to internal registry and not every image can be docker pulled. Additionally, with current IS implementation there's no single point of docker pull, any IS can point multiple docker registries and this is the way we're going to.